package.json

```diff
@@ -19,6 +19,7 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "ci": "yarn install --ignore-engines --frozen-lockfile",
```

Comment:
Is ignore engines still needed on node 14? And do we want to adjust the engines below?

Reply:
For now I'm keeping it all similar across all projects, but will remove in future.
From memory there is something that causes an issue (requires newer node) ...

`ember-cli-update`/`ember` declare the engine preference, so I'm keeping it where ember _believes_ the engines should be :)

Cheers man!